projects/common/src/main/java/dan200/computercraft/shared/pocket/peripherals/PocketNFCPeripheral.java

```diff
@@ -0,0 +1,30 @@
+package dan200.computercraft.shared.pocket.peripherals;
+
+import dan200.computercraft.api.peripheral.IPeripheral;
+import dan200.computercraft.api.pocket.IPocketAccess;
+
+import javax.annotation.Nullable;
+
+public class PocketNFCPeripheral extends PocketModemPeripheral {
+
+    public PocketNFCPeripheral(IPocketAccess access) {
+        super(false, access);
+    }
+
+    public void update(IPocketAccess access) {
+        setLocation(access);
+
+        var state = getModemState();
+        if (state.pollChanged()) access.setLight(state.isOpen() ? 0xf39d20 : -1);
```

Comment:
Traditionally only main peripherals set light status. This might look odd if this one and back peripheral will both try to set different colours.

Reply:
Good point, not sure how to address it though. Color blending? Blinking between the two colors?

Reply:
5c67b6c is my attempt at addressing this, the light slowly cycles between the two active colors